modules/imaging_uploader/README.md

```diff
@@ -53,9 +53,29 @@ For a successful upload:
 
 #### Module Permission
 
-The imaging uploader module uses one permission called `imaging_uploader` that 
-is necessary to have access to the module and gives the user the ability to 
-upload and browse all scans uploaded to the database.
+*In the interest of backwards compatibility, permission behaviour varies slightly 
+based on the `useAdvancedPermissions` configuration*
+
+Any of the following permissions grants access to the module.
+
+`imaging_uploader_allsites`: 
```

Comment:
I think it would be a good idea to mention somewhere that you _always_ have access to the scans _you_ uploaded.

Reply:
its in the test plan... not sure where itwould fit in the readme? any sugestions ?

Reply:
I would put the comment at the very end of the `Module Permission` section.